pkg/policy/mapstate.go

```diff
@@ -181,14 +226,24 @@ func (keys MapState) AllowAllIdentities(ingress, egress bool) {
 				labels.NewLabel(LabelKeyPolicyDerivedFrom, LabelAllowAnyEgress, labels.LabelSourceReserved),
 			},
 		}
-		keys[keyToAdd] = NewMapStateEntry(derivedFrom, false)
+		keys[keyToAdd] = NewMapStateEntry(derivedFrom, false, false)
 	}
 }
+func (pms MapState) GetIdentities(log *logrus.Logger) (ingIdentities, egIdentities []int64) {
+	return pms.getIdentities(log, false)
+}
+
+func (pms MapState) GetDenyIdentities(log *logrus.Logger) (ingIdentities, egIdentities []int64) {
+	return pms.getIdentities(log, true)
+}
```

Comment:
You could simply keep `GetIdentities` and add a boolean parameter as `isDeny`, which removes the need for an extra function `GetDenyIdentities`. If there are many calls to `GetIdentities`, then this might not be worth it.

Reply:
I think it looks explicit the current way